packages/graphiql/src/i18n.ts

```diff
@@ -0,0 +1,90 @@
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+
+const enTranslations = require('./locales/en/translation.json');
```

Comment:
are we able to use `import enTranslations from` or `import()` here for better tree shaking?